salt/modules/win_task.py

```diff
@@ -1919,110 +1938,132 @@ def add_trigger(name=None,
-            produces a weekly schedule. An interval of 2 produces an every-other
-            week schedule. If no interval is specified, 1 is used. Valid entries
-            are 1 - 52.
+        The task will run weekly.
+
+            weeks_interval (int):
+                The interval between weeks in the schedule. An interval of 1
+                produces a weekly schedule. An interval of 2 produces an
+                every-other week schedule. If no interval is specified, 1 is
+                used. Valid entries are 1 - 52.
 
-        days_of_week (list):
-            Sets the days of the week on which the task runs. Should be a list.
-            ie: ``['Monday','Wednesday','Friday']``. Valid entries are the names
-            of the days of the week.
+            days_of_week (list):
+                Sets the days of the week on which the task runs. Should be a
+                list. ie: ``['Monday','Wednesday','Friday']``. Valid entries are
+                the names of the days of the week.
 
     *Monthly*
 
-        months_of_year (list):
-            Sets the months of the year during which the task runs. Should be a
-            list. ie: ``['January','July']``. Valid entries are the full names
-            of all the months.
+        The task will run monthly.
 
-        days_of_month (list):
-            Sets the days of the month during which the task runs. Should be a
-            list. ie: ``[1, 15, 'Last']``. Options are all days of the month 1 -
-            31 and the word 'Last' to indicate the last day of the month.
+            months_of_year (list):
+                Sets the months of the year during which the task runs. Should
+                be a list. ie: ``['January','July']``. Valid entries are the
+                full names of all the months.
 
-        last_day_of_month (bool):
-            Boolean value that indicates that the task runs on the last day of
-            the month regardless of the actual date of that day.
+            days_of_month (list):
+                Sets the days of the month during which the task runs. Should be
+                a list. ie: ``[1, 15, 'Last']``. Options are all days of the
+                month 1 - 31 and the word 'Last' to indicate the last day of the
+                month.
 
-        .. note::
-            You can set the task to run on the last day of the month by either
-            including the word 'Last' in the list of days, or setting the
-            parameter 'last_day_of_month` equal to True.
+            last_day_of_month (bool):
+                Boolean value that indicates that the task runs on the last day
+                of the month regardless of the actual date of that day.
+
+                .. note::
+                    You can set the task to run on the last day of the month by
+                    either including the word 'Last' in the list of days, or
+                    setting the parameter 'last_day_of_month` equal to True.
```

Comment:
Inconsistent quoting for `last_day_of_month`

Reply:
Thanks! Got it.